.gitignore

```diff
@@ -31,3 +31,4 @@ web/javascripts
 web/js
 web/stylesheets
 tests/behat/behat.yml
+app/cache
```

Comment:
This shouldn't be necessary any more since the cache is now stored in `var/cache`. If you've got old artifacts in `app/cache` you should just be able to delete them and they won't reappear.

Reply:
removed